### DISCUSSION TITLE
Why am I still wallet zero XCH [help]

### DISCUSSION THREAD

**Discussion by hmz06967** [Chia-Network/chia-blockchain#3607] [General]
Hi there, 
I've been farming for a few days. What is the min chia reward I will see in my wallet? still 0 for example 0.05 XCH vs

**Comment by yannickpschroeder** [comment]
It works like a diet. You don't eat much for a day and the next day you expect to be thinner. But its a process my friend and takes some time ;)

**Comment by hmz06967** [comment]
Well,
But I'm just saying, is there a minimum limit? 
Like Min 1 chia coin.
When I'm on a diet, for example, I say my weight limit is going to be this, and when I get to that figure, I quit😂

**Comment by henricavalcante** [comment]
Since you are not farming in a pool, you will bet for a whole prize, 2 XCH, each 10 seconds people are winning, so you have to wait your lucky time.

They said that pool protocol is one of the priorities now, so it will b better in a few days when we have official pools to farm together.

**Comment by Muffexx** [comment]
Hello @hmz06967,

when you win, you will always win 2 coins. The higher the total size of your plots, the higher is your chance to win. The estimated time to win is visible in your farming overview. But keep in mind that this estimate depends heavily on the network growth (the greater the network the lower your chance to win). 

regards
muffex

**Comment by hmz06967** [comment]
Thank you for answer guys good farmingg

**Comment by 1971192997** [comment]
hi,  what is the decimal for xch?     8 or 12?

**Reply by hmz06967** [reply]
Whats decimal number?

**Reply by Nikolaj-K** [reply]
@hmz06967 Presumably he means the number of decimals of the coin currency. 
E.g. 1 bitcoin is 10^8 sats, which is the actual `int` unit in which amounts are stored (the dot is fictional)

**Comment by crazyhei82** [comment]
i have 50 plot still have 0 xch .

i want to cry .

am i not lucky ?

if you win the 1 xch all the plot will disappear? or still heer to wait another lucky draw?

**Reply by virgiliofilhos** [reply]
I am in the same situation ...
I do not know what to do...

wanted to see the config.yaml of someone who has already received some currency

**Reply by Team4N6** [reply]
You keep your plots after you win a block. So they're always there. However, plots becomes less valuable over time because the network size keeps growing. (i.e. the chance of any one plot winning decreases).

**Comment by virgiliofilhos** [comment]
> Hi there,
> I've been farming for a few days. What is the min chia reward I will see in my wallet? still 0 for example 0.05 XCH vs

I am in the same situation ...
I do not know what to do...

wanted to see the config.yaml of someone who has already received some currency

**Comment by Team4N6** [comment]
As others have said, with the extreme network growth, farming small groups of plots (<500) has become untenable. Pooling will help this when it's released, so at least you could get small rewards for your plots. Even with 500 plots, today you're likely to win once a week. With the current network growth, by next week that will be cut down significantly. If you have 50 plots, it estimates 4 months, but long before you win that will increase to years.

**Reply by hmz06967** [reply]
Hi, chiacalculator.com As he said, I earn after 4 months and the growth rate is 15%, but how true is that?

**Reply by pluto2021** [reply]
That calculator is misleading. That time to earn is probably at best 50% chance. That chance would get eroded quickly as the network growth outpace any solo farmer. Realistically, it may need to multiply by 3x or 4x as shown in the GUI. 4month would turn into 12-16months if the network were NOT to grow!  If this network space growth is not to slow down, there could be a large number of solo famers that might never win for their entire life. The exponential growth of networks space means there are more and more plotting power joining. This is next to impossible for any solo farmer to compete. We are really crying for the official pooling.  Otherwise, we can call this as the Chia Lotto lol.

**Reply by hmz06967** [reply]
Is winning a chance, or are we playing the lottery? Won't solving a Block make us money? We need a chance to untie the block? They say the pool will be here soon.

**Comment by anderzheng** [comment]
500plots with 0 xch...

**Reply by hmz06967** [reply]
Why :(

**Reply by anderzheng** [reply]
I'm not a lucky dog...

**Reply by BigUncle** [reply]
I'm another unlucky dog~~~

**Comment by rodollfomartins** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/83837213/117491882-3021ea80-af47-11eb-923b-22bb0268e022.png)
posted in the group

**Reply by Muffexx** [reply]
Lucky one

**Reply by Team4N6** [reply]
Nice!

**Reply by Coiot3** [reply]
Poderia dar uma ajuda? No meu caso o programa não aparece a opção "Negociar". Já tenho um plot com essa extensão: plot-k32-2021-05-06-12-39-35 plot.2.tmp Está com o tmp no final. Quando ele torna-se somente  plot ?

![Screenshot_46](https://user-images.githubusercontent.com/83840407/117504244-88151d00-af58-11eb-806e-734e92d95934.png)

![Screenshot_2](https://user-images.githubusercontent.com/83840407/117501805-f3f58680-af54-11eb-93f0-d8fc6d380105.png)
![Screenshot_44](https://user-images.githubusercontent.com/83840407/117501814-fa83fe00-af54-11eb-8d0d-0efe87a8fa11.png)
![Screenshot_45](https://user-images.githubusercontent.com/83840407/117501826-fd7eee80-af54-11eb-85c0-30f790345d0e.png)
